tests/foreman/virtwho/test_cli_virtwho.py

```diff
@@ -274,19 +279,40 @@ def test_positive_filter_option(self):
             'id': vhd['id'],
             'filtering-mode': 'whitelist',
             'whitelist': regex,
+            'filter-host-parents': regex
```

Comment:
`filter-host-parents` and `exclude-host-parents` are only used by esx mode

Reply:
ohh, I forgot it, thanks